sql/parser/union.go

```diff
@@ -29,15 +29,20 @@ import "fmt"
 type Union struct {
 	Type        string
 	Left, Right SelectStatement
+	All         bool
 }
 
 // Union.Type
 const (
-	astUnion     = "UNION"
-	astExcept    = "EXCEPT"
-	astIntersect = "INTERSECT"
+	astUnion     = " UNION"
```

Comment:
Not sure why you changed this. Not that it matters. When we get around to implementing `Union` the strings will need to be replaced with enumerated values.